autoware_map_msgs/srv/GetDifferentialLanelet2Map.srv

```diff
@@ -0,0 +1,9 @@
+# Osm file ids which are selected to be loaded
+int32[] osm_file_ids
```

Comment:
The name osm is not often used in Autoware. How about the following?
- vector_map_file_ids
- file_ids
- lanelet2_file_ids

Reply:
I just made this similar with point cloud metadata files but this with this way, it look better. Updated.